pages/index.tsx

```diff
@@ -66,45 +69,83 @@ export default function Home() {
         />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <section className={styles.hero}>
-        <h1 className={styles.title}>Awesome Kurds</h1>
-        <p className={styles.slogan}>
-          Meet {awesomeKurds.kurds.length} awesome Kurds.
-        </p>
-        <div className={styles.CTA}>
+      <Layout
+        style={{
+          backgroundColor: "transparent",
+          height: "100vh",
+        }}
+      >
+        <Content
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "center",
+          }}
+        >
+          <Title>Awesome Kurds</Title>
+          <Text type="secondary">
```

Comment:
This text is appearing too small, in my opinion.

Reply:
I agree, but if you want to make it bigger I guess you need to use vanilla css. I wanted not to use `style={{...}}` and `className={styles.STYLE_NAME}` as much as I could.

Reply:
Maybe use another variant of `Text`? There's also no issue with `style={{...}}` imho.

Reply:
There is not any variant that supports bigger text. I think for this issue if you use `style=...` and make the font bigger, then you need to make it smaller on small screens and you can achieve this in `css` which then you need to have `className={styles....}` and make the code messier. By far this is what I have thought of, feel free to commit to this pr tho.